src/data/real/ennreal.lean

```diff
@@ -838,6 +838,9 @@ end
 lemma mul_sub (h : 0 < c → c < b → a ≠ ∞) : a * (b - c) = a * b - a * c :=
 by { simp only [mul_comm a], exact sub_mul h }
 
+lemma antitone_const_sub {c : ℝ≥0∞} : antitone (λ (x : ℝ≥0∞), c - x) :=
```

Comment:
instead of having this just on ennreal, can you give it under the same assumptions as tsub_le_tsub?